src/table-definition/XhtmlTableDefinition.js

```diff
@@ -139,14 +144,17 @@ define([
 			// Creates
 			createCellNodeStrategy: createCreateCellNodeStrategy(namespaceURI, 'td'),
 			createRowStrategy: createCreateRowStrategy(namespaceURI, 'tr'),
+			createColumnSpecificationNodeStrategy: createCreateColumnSpecificationNodeStrategy(namespaceURI, 'col', './*[self::' + thead + ' or self::' + tbody + ']'),
 
 			// Specification
 			getTableSpecificationStrategies: useBorders ? [
 					getSpecificationValueStrategies.createGetValueAsBooleanStrategy('borders', './@border = "1"')
 				] : [],
 
 			getColumnSpecificationStrategies: [
-					getSpecificationValueStrategies.createGetValueAsStringStrategy('columnWidth', '"1*"')
+					getSpecificationValueStrategies.createGetValueAsStringStrategy('columnWidth', '"1*"'),
+					getSpecificationValueStrategies.createGetValueAsStringStrategy('horizontalAlignment', './@align'),
+					getSpecificationValueStrategies.createGetValueAsStringStrategy('verticalAlignment', './@valign')
```

Comment:
I do not think this is even needed. Some of the purely visual aspects of parts of a table are saved in our abstract table model (the thing we are declaring here) because of backwards compatibility reasons. For instance, we have a setCellData operation with which you can set a number of properties which are then synced back to the XML. I think the outputclass approach you chose in the PR is even preferable to using celldata because it is better in describing where which data should be saved in the XML.

By the way, the next Fonto release is scheduled to contain a new alignment property for just about every family. Using this property, the need for custom CSS + outputclasses is removed. We can then even think about configuring alignment for cells in this package.

Reply:
Hi @DrRataplan,
hm, sorry but I can't find the `setCellData` function in the [documentation](http://documentation.fontoxml.com/api/latest/function-12423104.html) (although it seems to be in the `fontoxml-table-flow` package). Is this an official API functionality?

Thanks for teasing me about the new Fonto update! Looking forward to it :)

Reply:
Ah, yes, `setCellData` is an operation which is scheduled to be made public API in the next release. It provides an abstraction over cell data in some circumstances and is used for cell borders in some apps.

For these purely visual aspects, the abstraction turned out to be confusing. I like your approach of directly working with XML way better!